tests/test_post_to_studio.py

```diff
@@ -157,7 +166,9 @@ def test_post_to_studio_skip_start_and_done_on_env_var(
         live.log_metric("foo", 1)
         live.next_step()
 
-    assert mocked_post.call_count == 2
+    call_types = [call.kwargs["json"]["type"] for call in mocked_post.call_args_list]
+    assert "start" not in call_types
+    assert "done" not in call_types
```

Comment:
Better to test that `start` and `done` events aren't called rather than call counts.